test/instrumentation/modules/kafkajs/fixtures/use-kafkajs-each-message.js

```diff
@@ -80,7 +78,7 @@ async function useKafkajsClient(kafkaClient, options) {
     messages: [
       { value: 'each message 1', headers: { foo: 'string' } },
       { value: 'each message 2', headers: { foo: Buffer.from('buffer') } },
-      { value: 'each message 3' },
+      { value: 'each message 3', headers: { auth: 'secret_token' } },
```

Comment:
```suggestion
      { value: 'each message 3', headers: { auth: 'this_is_a_secret' } },
```

super-nerd-crazy-nit: To avoid a grep hit with `secret_token` in lib/config/schema.js.